### DISCUSSION TITLE
这是干嘛的

### DISCUSSION THREAD

**Discussion by hhbible** [TomSchimansky/CustomTkinter#1917] [Q&A]
Is this a library or a finished code?

**Comment by Akascape** [comment]
@hhbible 
Library, this is python based modern-graphical interface toolkit built in top of native tkinter gui framework. All the sub widgets of customtkinter are built with the help of tkinter canvas (with lots of modifications). So every widget is different from the basic tkinter widgets. People use this library to build their own modern desktop applications. Start your journey from the documentation page: 
https://customtkinter.tomschimansky.com/

More awesome tkinter based libraries can be found here: https://github.com/Akascape/tkinter-toolkit
